src/Lessons-api.Data/Repositories/StudentRepository.cs

```diff
@@ -39,28 +40,31 @@ public async Task<StudentEntity> AddStudent(StudentEntity studentEntity)
             return studentEntity;
         }
 
-        public async Task<StudentEntity> UpdateStudent(int id, StudentEntity studentEntity)
+        public async Task<StudentEntity> UpdateStudent(int id, UserEntity userEntity)
         {
-            var updatedStudent = await _lessonsContext.Students.FindAsync(id);
+            var updatedStudent = await _lessonsContext.Students.Include(s => s.User).SingleOrDefaultAsync(s => s.Id == id);
```

Comment:
Change this to FirstOrDefaultAsync

![image](https://user-images.githubusercontent.com/64907389/128723910-174c7913-77eb-4852-b8e5-78aa23e7ee7e.png)